lib/improver/blending/blend_across_adjacent_points.py

```diff
@@ -103,7 +103,8 @@ def __init__(self, coord, central_point, parameter_units, width,
         # Set up the blending function, based on whether weighted blending or
         # maximum probabilities are needed.
         self.BlendingPlugin = (
-            WeightedBlendAcrossWholeDimension(coord, weighting_mode))
+            WeightedBlendAcrossWholeDimension(coord, weighting_mode,
+                                              timeblending=True))
 
```

Comment:
Not part of your changes I know, but it seems there's an unused import at the top of this module:

`from improver.utilities.cube_manipulation import concatenate_cubes`